packages/storage/src/platform/base64.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This file is only used under ts-node.
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const platform = require(`./${process.env.TEST_PLATFORM ?? 'node'}/base64`);
```

Comment:
Where is `TEST_PLATFORM` set? Do we always default to `node` currently?

Reply:
Actually seems it's unnecessary to set it, because we default to `browser` in the [webpack.config](https://github.com/firebase/firebase-js-sdk/blob/master/config/webpack.test.js#L106) if `TEST_PLATFORM` is not set.

I think it also means we can just import `./node/base64` here since this file is only used for node tests as you pointed out in the comment.

Reply:
Sounds good. I simplified it.